dcmdump/src/main.rs

```diff
@@ -263,81 +322,93 @@ where
     D: DataDictionary,
 {
     let indent: String = std::iter::repeat(' ').take((depth * 2) as usize).collect();
-    writeln!(to, "{}(FFFE,E000) na Item", indent)?;
+    writeln!(
```

Comment:
The indentation should go before the tag. Things look weird when dumping sequence items.

```none
(0018,6011) SequenceOfUltrasoundRegions         SQ (2 Items)
(FFFE,E000)     na Item
      (0018,6012) RegionSpatialFormat                 US (1,  2 bytes): 1
      (0018,6014) RegionDataType                      US (1,  2 bytes): 1
      (0018,6016) RegionFlags                         UL (1,  4 bytes): 3
(FFFE,E00D)     ItemDelimitationItem
(FFFE,E000)     na Item
```